docs/source/cli/sawtooth.rst

```diff
@@ -112,6 +112,13 @@ sawtooth batch show
    :language: console
    :linenos:
 
+sawtooth batch-status
+===================
```

Comment:
The doc build is breaking on this line (the equals underline needs to be as long as the title on the previous line).